tools/config2kv.py

```diff
@@ -60,9 +60,9 @@
 
 
 def _read_config_settings(conf_file):
-    sys_config = conf_file = os.environ["MSTICPYCONFIG"]
-    if not conf_file:
-        conf_file = sys_config
+    # sys_config = os.environ["MSTICPYCONFIG"]
```

Comment:
why are these commented out?

Reply:
This script wasn't working - I had to comment out some lines to get it to work and forgot to make proper changes. Ill do that now.